publication-commons/src/main/java/no/unit/nva/publication/permissions/publication/grant/CuratorGrantStrategy.java

```diff
@@ -36,27 +36,23 @@ public boolean allowsAction(PublicationOperation permission) {
                  SUPPORT_REQUEST_CREATE,
                  DOI_REQUEST_CREATE,
                  PUBLISHING_REQUEST_CREATE,
-                 UPLOAD_FILE -> canManageStandardResources();
-            case UNPUBLISH -> canManagePublishingRequests() && isPublished();
+                 UPLOAD_FILE -> isCurator();
+            case UNPUBLISH -> hasApprovedFiles() ? canApproveFiles() && isPublished() : isCurator();
```

Comment:
"Curators" should be able to UNPUBLISH when publication does not have approved files. 
Updating from `canManagePublishingRequest` to `canApproveFiles` so we allow "degree curator" to UNPUBLISH degree, and "curator" to UNPUBLISH non degree.